include/dashbls/schemes.hpp

```diff
@@ -186,7 +187,7 @@ class AugSchemeMPL : public CoreMPL {
                          const G2Element& signature) override;
 };
 
-class PopSchemeMPL : public CoreMPL {
+class PopSchemeMPL final : public CoreMPL {
```

Comment:
missing `class LegacySchemeMPL final : public CoreMPL {`
that is not presented in Chia's library.

Reply:
Thanks! added in 72f0ad2